lib/iris/fileformats/grib/__init__.py

```diff
@@ -1053,38 +828,24 @@ def save_pairs_from_cube(cube):
         yield (slice2D, grib_message)
 
 
-def as_messages(cube):
-    """
-    .. deprecated:: 1.10
```

Comment:
yes, ready to remove;

these were misnamed and then renamed, hence the deprecation; they're part of the v2 removal